pkg/kubectl/cmd/logs.go

```diff
@@ -51,6 +52,8 @@ var (
 
 		# Show all logs from pod nginx written in the last hour
 		kubectl logs --since=1h nginx`)
+
```

Comment:
can you update the comment here to give some examples of what this supports. Like kubectl logs -lapp=nginx. I am guessing this will only select pods with that label. Also what happens if the pod contains multiple containers, does it show logs for all container as well ?